conans/client/generators/virtualenv_python.py

```diff
@@ -0,0 +1,16 @@
+from conans.client.generators.virtualrunenv import VirtualRunEnvGenerator
+
+
+def _python_paths(deps_env_info):
+    result = set()
+    for dep in deps_env_info.deps:
```

Comment:
Not sure if we should access directly to the aggregated value of `deps_env_info.get("PYTHONPATH")`. Have you tried?

Reply:
`self.deps_env_info.vars.get("PYTHONPATH", [])` works.